packages/synthetix-main/contracts/interfaces/IFundVault.sol

```diff
@@ -0,0 +1,78 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IFundVaultStorage.sol";
+
+/// @title Module for managing funds and assignments per account
+interface IFundVault is IFundVaultStorage {
+    // /// @notice external access to rebalanceMarkets
+    // function rebalanceMarkets(uint fundId) external; // TODO Maybe is internal
+
+    /// @notice delegates (creates, adjust or remove a delegation) collateral from an account
+    function delegateCollateral(
+        uint fundId,
+        uint accountId,
+        address collateralType,
+        uint amount,
+        uint leverage
+    ) external;
+
+    /// @notice mints sUSD for a fund/account from a collateralType. if CRatio is valid
+    function mintsUSD(
```

Comment:
im realizing now it might just be better to call this `withdrawLoan` or something like that, `sUSD` is a terrible thing to have in function names unfortunately

Reply:
I'll keep that way until we find/choose a better option. We were discussing it a little bit with Noah and couldn't find a better option.